setup/terraform/resources/common.sh

```diff
@@ -1329,6 +1329,8 @@ function deploy_cluster_prereqs() {
   yum_install nodejs gcc-c++ make shellinabox mosquitto transmission-cli rh-python38 rh-python38-python-devel httpd
   # Below is needed for secure clusters (required by Impyla)
   yum_install cyrus-sasl-md5 cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-devel
+  # SSB-LB-UI needs libssl.so.1.1
```

Comment:
@ylysn Do you know a way to reproduce the error that led you to add this?
I'm curious what may have caused this, since I never hit this problem.
Which stack(s) were you testing?